nevergrad/benchmark/experiments.py

```diff
@@ -929,9 +929,12 @@ class PackedFunctions(ExperimentFunction):
 
     def __init__(self, functions: tp.List[ArtificialFunction], upper_bounds: np.ndarray) -> None:
         self._functions = functions
+        assert len(functions) > 0
         self._upper_bounds = upper_bounds
         self.multiobjective = MultiobjectiveFunction(self._mo, upper_bounds)
         super().__init__(self.multiobjective, self._functions[0].parametrization)
+        self._parametrization.descriptors.not_manyobjective = len(functions) < 4
+        self._parametrization.descriptors.not_multiobjective = len(functions) == 1
```

Comment:
monoobjective?

Reply:
Ah yes that's clear better than my not_multi :-)